config/config.exs

```diff
@@ -18,6 +18,18 @@ config :tilex, TilexWeb.Endpoint,
            adapter: Phoenix.PubSub.PG2],
   http: [protocol_options: [max_request_line_length: 8192, max_header_value_length: 8192]]
 
+# Provide reasonable default for configuration options
+config :tilex, :page_size, 5
+config :tilex, :auth_controller, AuthController
+config :tilex, :slack_notifier, Tilex.Slack
+config :tilex, :twitter_notifier, Tilex.Twitter
+config :tilex, :organization_name, System.get_env("ORGANIZATION_NAME")
+config :tilex, :ga_identifier, System.get_env("GA_IDENTIFIER")
```

Comment:
The changes to this file are great. The only thing I'd like to see is the `GA_IDENTIFIER`moved back to the production config file. We load the JS only if this variable is set, and we only want Google Analytics in production.

Setting it in the default config has led us to accidentally run Google Analytics in dev and test before, as @tmock12 noted.